server/db/dbManager.ts

```diff
@@ -288,11 +325,36 @@ export class UserManager {
 			username,
 			password: hashedPassword,
 			upvotes: {},
-			downvotes: {}
+			downvotes: {},
+			reputation: 0,
+			numberOfPosts: 0
 		});
 		return 'success';
 	}
 
+	async changeIconPath(username: string, path: string): Promise<string> {
```

Comment:
Perhaps, `update` suits better?
```suggestion
	async updateIconPath(username: string, path: string): Promise<string> {
```